assets/src/dashboard/app/views/shared/storyGridView.js

```diff
@@ -115,14 +115,10 @@ const StoryGridView = ({
       {filteredStories.map((story) => (
         <CardGridItem key={story.id} isTemplate={isTemplate}>
           <CardPreviewContainer
-            centerAction={
-              isTemplate
-                ? undefined
-                : {
-                    targetAction: story.centerTargetAction,
-                    label: centerActionLabel,
-                  }
-            }
+            centerAction={{
+              targetAction: story.centerTargetAction,
+              label: centerActionLabel,
+            }}
```

Comment:
I think with this we get the Preview button again on the templates. I wasn't sure if we wanted that or not with the new designs.

Reply:
It just enables a middle link that we can call whatever we want.  I named it "View" for templates and it links to the template detail page which I think is part of the new designs too.
![image](https://user-images.githubusercontent.com/40646372/80771609-d94e2d80-8b08-11ea-9cba-346ded58e292.png)